guides/common/modules/proc_configuring-dns-nsupdate.adoc

```diff
@@ -0,0 +1,19 @@
+[id="configuring_dns_nsupdate_{context}"]
+= Configuring dns_nsupdate
+
+The _dns_nsupdate_ DNS provider manages DNS records using nsupdate.
+It can be used with any https://www.rfc-editor.org/rfc/rfc2136[RFC2136] compatible DNS server, but by default ISC BIND is installed.
+For installation without ISC BIND, see xref:configuring-external-dns_{context}[].
+
+.Procedure
+* You can use `{foreman-installer}` to configure `dns_nsupdate`:
```

Comment:
```suggestion
* Use `{foreman-installer}` to configure `dns_nsupdate`:
```
or even
```suggestion
* Configure `dns_nsupdate`:
```
because users can see that they are using the installer, no need to tell them twice. Instead, you _could_ provide more details or a bit of context about the actual command example, but then again, that's all included in --help already so this is probably just fine as is.

Reply:
I think it's an often repeating thing. We have many procedures that are essentially a single step that is "run the installer with these arguments". I never quite know what's best because that header in the procedure almost feels redundant.